content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,169 @@
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the impacted rules based on the current pipeline context (branch, repository).
+The rules that are retrieved vary based on the pipeline context, the [rules scopes](#rule-scope) and eventual [custom scopes](#custom-scope).
+2. Evaluates all the impacted rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger"><strong>Note:</strong> for the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+The command requires the `DATADOG_API_KEY` and `DATADOG_APP_KEY` environment variables to point to your [Datadog API Key][5]
+and [Datadog Application Key][6]. Also, you need to set the `DD_BETA_COMMANDS_ENABLED` environment
+variable as `true`. Optionally, you can specify the `DATADOG_SITE` environment variable to point to a specific datadog site.
+The default site is US1 (datadoghq.com).
+
+{{< code-block lang="shell" >}}
+DD_BETA_COMMANDS_ENABLED=true DATADOG_API_KEY=<API_KEY> DATADOG_APP_KEY=<APP_KEY> datadog-ci gate evaluate
+{{< /code-block >}}
+
+The behavior of the command can be modified using the following flags:
+- **--fail-on-empty**: when this flag is specified, the command fails if no matching rules were found in Datadog
+based on the current command scope. By default, the command succeeds.
+- **--fail-if-unavailable**: when this flag is specified, the command fails if one or more rules could not be evaluated.
+By default, the command succeeds.
+- **--no-wait**: by default, the command waits a certain amount of time for the events (tests, static analysis violations) to arrive to Datadog.
+This step is important as it makes sure that the events are queryable in Datadog before the rules are executed,
+avoiding incorrect evaluation. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is
+called several minutes after the related events are sent to Datadog, you could skip this waiting time by specifying the `--no-wait` flag.
+Note that, if used incorrectly, this flag might result in inaccurate rule evaluations.
+
+[Custom scopes](#custom-scope) can be added by using the following option one or more times: **--scope**:
+
+{{< code-block lang="shell" >}}
+datadog-ci gate evaluate --scope team:backend --scope team:frontend
+{{< /code-block >}}
+
+The command logs can be examined to understand what was the overall gate evaluation status, along with information
+about all the rules that were evaluated.
+
+{{< img src="ci/datadog_ci_gate_evaluate_logs.png" alt="Datadog-ci gate evaluate logs" style="width:90%;">}}
```

Comment:
❓ Could we include this as a code block, so actual text, instead of an image? It's not very readable IMO.

Reply:
Mmm yeah that could be done, although a nice thing of images is that they can be zoomed in easily, whereas the code can't. It seems that we are just showing something here, they don't need to use it anywhere, so maybe the code won't be the right format to represent this (e.g. the copy button won't make much sense)